cmd/docker-nv2/push.go

```diff
@@ -37,29 +37,42 @@ func pushImage(ctx *cli.Context) error {
 	}
 
 	fmt.Println("Pushing signature")
-	sigPath := config.SignaturePath(desc.Digest)
-	sig, err := ioutil.ReadFile(sigPath)
+	sigDigests, err := config.SignatureDigests(desc.Digest)
 	if err != nil {
-		if os.IsNotExist(err) {
-			return errors.New("signature not found")
-		}
 		return err
 	}
+	if len(sigDigests) == 0 {
+		return errors.New("no signatures found")
+	}
 
 	client, err := docker.GetSignatureRepository(ctx.Context, ctx.Args().First())
 	if err != nil {
 		return err
 	}
-	sigDesc, err := client.Put(ctx.Context, sig)
-	if err != nil {
-		return err
-	}
+	pushSignature := func(sigDigest digest.Digest) error {
+		sigPath := config.SignaturePath(desc.Digest, sigDigest)
+		sig, err := ioutil.ReadFile(sigPath)
```

Comment:
ioutil should not be used anymore as it is deprecated since go 1.16. 

Recommend is to use the replacements from the `os` package.

https://golang.org/doc/go1.16#ioutil

Reply:
Good catch as this code is just migrated from `1.14 to 1.16`. I've resolved it and may scan other code later.